src/jquery.i18n.fallbacks.js

```diff
@@ -1,4 +1,4 @@
-/*!
+/* !
```

Comment:
It's not about what I want; I cannot decide conventions alone. /*! is required by jshint (or jscs, I don't remember right now) because otherwise it would complain about @license tags being unknown tags. If it is no longer deemed necessary, then ! should be removed.